CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 0.66.2      2025-06-25
+
+* Amazon Aurora: Add support for Postgres 17
+  - The collector now supports Amazon Aurora with Postgres 17
+  - Previously, Amazon Aurora users on Postgres 17 were unable to collect query
+    statistics due to a column "blk_read_time" does not exist error, caused by
+    changes to pg_stat_statements
```

Comment:
Technically that was a change in aurora_stat_plans I believe (inspired by a change in pg_stat_statements, but functionally separate)

Reply:
yeah I'll actually drop "caused by changes to pg_stat_statements. I wanted to include the error message (for searchability) so I'll leave that part still